esprima.js

```diff
@@ -1948,11 +1948,12 @@
             return this;
         },
 
-        finishTryStatement: function (block, guardedHandlers, handlers, finalizer) {
+        finishTryStatement: function (block, handler, finalizer) {
             this.type = Syntax.TryStatement;
             this.block = block;
-            this.guardedHandlers = guardedHandlers;
-            this.handlers = handlers;
+            this.guardedHandlers = [];
+            this.handlers = handler ? [ handler ] : [];
```

Comment:
Giving a warning right now is too premature anyway. We shall try to adjust popular downstream project to track `handler` instead and see if further migration warrants giving the warning or not.